android/app/src/main/java/app/organicmaps/MwmApplication.java

```diff
@@ -296,13 +296,15 @@ private native void nativeInitPlatform(String apkPath, String writablePath, Stri
     @Override
     public void onStart(@NonNull LifecycleOwner owner)
     {
-      MwmApplication.this.onForeground();
+      if (mDisplayManager.isDeviceDisplayUsed())
+        MwmApplication.this.onForeground();
```

Comment:
This is extremely confusing. This application-wide onBackground()/onForeground() calls are disabled in Auto mode?

Reply:
> This application-wide onBackground()/onForeground() calls are disabled in Auto mode?

No.

The problem is that `onBackground` called `LocationHelper.INSTANCE.stop();`
I covered `LocationHelper.INSTANCE.stop();` with this check and that's enough. It works fine.

Reply:
ok, I will check how to refactor it later.